lib/marked.js

```diff
@@ -875,7 +912,7 @@ Renderer.prototype.link = function(href, title, text) {
     } catch (e) {
       return '';
     }
-    if (prot.indexOf('javascript:') === 0 || prot.indexOf('vbscript:') === 0 || prot.indexOf('data:') === 0) {
+    if (prot.indexOf('javascript:') === 0 || prot.indexOf('vbscript:') === 0) {
```

Comment:
Why would you remove this XSS protection?

Reply:
By no means would I remove this. I will revert it.